README.md

```diff
@@ -3,6 +3,7 @@
 An ESLint plugin for linting ESLint plugins. Rules written in CJS, ESM, and TypeScript are all supported.
 
 <!-- vscode-markdown-toc -->
+
```

Comment:
Looks like the table of contents needs to be updated which will be automatic if you have this extension:

Name: Markdown All in One
Id: yzhang.markdown-all-in-one
Description: All you need to write Markdown (keyboard shortcuts, table of contents, auto preview and more)
Version: 3.6.2
Publisher: Yu Zhang
VS Marketplace Link: https://marketplace.visualstudio.com/items?itemName=yzhang.markdown-all-in-one

Reply:
Err, which command should I run?

Reply:
Normally it just updates it for you when you save the file.

Reply:
Well, ah, this is what it updated me to.

Reply:
Can you add `<!-- omit from toc -->` to the end of the title line?

https://github.com/yzhang-gh/vscode-markdown#table-of-contents

Reply:
That did it, thanks!